### PR TITLE
feat(ui) Make loading messages dynamic

### PR DESCRIPTION
**Pull request by markstory**
Customers often point out that they like our loading messages. Add some surprise and delight by having a selection of phrases to choose from. I'd appreciate any other options as well.

### CODE REVIEW

File: src/sentry/templatetags/sentry_helpers.py [hunk 1]
```diff
@@ -109,6 +110,18 @@ def org_url(organization, path, query=None, fragment=None) -> str:
     return organization.absolute_url(path, query=query, fragment=fragment)
 
 
+@register.simple_tag
+def loading_message():
+    options = [
+        "Please wait while we load an obnoxious amount of JavaScript.",
+        "Downloading a dizzying amount of JavaScript.",
+        "Downloading a dizzying amount of JavaScript.",
```
Comment (dashed) [inline src/sentry/templatetags/sentry_helpers.py:118]:
```suggestion
```

File: src/sentry/templatetags/sentry_helpers.py [hunk 2]
```diff
@@ -109,6 +110,18 @@ def org_url(organization, path, query=None, fragment=None) -> str:
...
+        "Reticulating splines.",
+        "Aquiring more vespene gas.",
```
Comment (dashed) [inline src/sentry/templatetags/sentry_helpers.py:120]:
```suggestion
        "Acquiring more vespene gas.",
```

**Review by dashed — APPROVED** [review]
Looks good to me.

**Review by vbro — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
super cool!

**Comment by gggritso** [comment]
For your consideration:

- Packaging package management management packages
- Awaiting solution to halting problem
- Collapsing wavefunctions
- `await watch(pot.boil())`
- Ironing out dents in universe

**Pull request merged**